README.md

```diff
@@ -133,8 +136,14 @@ And listen to interfaces `lo` and `docker0`.
 *step 3* Configure docker to use the `docker0` as DNS server
 
     # For systemd users (Fedora and recent Ubuntu versions) :
-    $ vim /lib/systemd/system/docker.service
-    # append the --bip="172.17.42.1/24" --dns="172.17.42.1" options to the ExecStart line
+    
+    $ vim /etc/systemd/system/docker.service
+    
+    # append
+    [Service]
+    ExecStart=
```

Comment:
I don't know why, but it don't work if you don't do it (specified like that in the docker doc)